### ISSUE TITLE
integrate ajax success with timeout functionality

### ISSUE THREAD

**Issue by danhammari** [t4t5/sweetalert#386]
I would like to use the sweet alert box to notify users of progress during an ajax request. I would trigger a sweet alert message and immediately kick off an ajax request. The sweet alert would show a message such as "processing request..." with a rotating spinner. When the ajax request returns with a success message, the rotating spinner would hide and the message would change to "request is complete" or something similar:

```javascript
swal({
    title: 'ajax request pending',
    text: 'processing request',
    showConfirmButton: false,
    closeOnConfirm: false,
    showSpinner: true,
    ajax: function(){
        url: 'https://domain.com/controller/method/action',
        type: post,
        dataType: json,
        data: { key: value },
        success: swal.confirm( response.message ),
        error: swal.confirm( response.message )
    }
},
function(){
    swal({
        title: 'ajax request finished',
        text: response.message,
        showSpinner: false,
        showConfirmButton: true,
        closeOnConfirm: true
});
```

**Comment by chriscct7** [comment]
Duplicate of https://github.com/t4t5/sweetalert/issues/302

**Comment by t4t5** [comment]
This is available in v1.1.0 with `showLoaderOnConfirm: true`!

**Comment by KimSpecht** [comment]
I cannot find any documentation for this feature either here or in the linked issue.
"showLoaderOnConfirm: true" does not start any calls on it's own for obvious reasons.
